common/src/main/java/org/vivecraft/mixin/client/renderer/GameRendererVRMixin.java

```diff
@@ -857,167 +887,6 @@ public Matrix4f getThirdPassProjectionMatrix() {
-		if (!this.minecraft.noRender) {
-			Window window = this.minecraft.getWindow();
-			Matrix4f matrix4f = Matrix4f.orthographic(0.0F, (float) ((double) window.getWidth() / window.getGuiScale()),
-					0.0F, (float) ((double) window.getHeight() / window.getGuiScale()), 1000.0F, 3000.0F);
-			RenderSystem.setProjectionMatrix(matrix4f);
-			PoseStack posestack = RenderSystem.getModelViewStack();
-			posestack.pushPose();
-			posestack.setIdentity();
-			posestack.translate(0.0D, 0.0D, -2000.0D);
-			RenderSystem.applyModelViewMatrix();
-			Lighting.setupFor3DItems();
-			PoseStack posestack1 = new PoseStack();
-
-			int i = (int) (this.minecraft.mouseHandler.xpos() * (double) this.minecraft.getWindow().getGuiScaledWidth()
-					/ (double) this.minecraft.getWindow().getScreenWidth());
-			int j = (int) (this.minecraft.mouseHandler.ypos() * (double) this.minecraft.getWindow().getGuiScaledHeight()
-					/ (double) this.minecraft.getWindow().getScreenHeight());
-
-
-			GameRendererVRMixin.DATA_HOLDER.pumpkineffect = 0.0F;
-
-			if (renderWorldIn && this.minecraft.level != null
-					&& (!this.minecraft.options.hideGui || this.minecraft.screen != null)) {
-				this.minecraft.getProfiler().popPush("gui");
-//				if (Reflector.ForgeIngameGui.exists()) {
-//					// RenderSystem.defaultAlphaFunc();
-//					Reflector.ForgeIngameGui_renderVignette.setValue(false);
-//					Reflector.ForgeIngameGui_renderPortal.setValue(false);
-//					Reflector.ForgeIngameGui_renderCrosshairs.setValue(false);
-//				}
-
-				// no thanks.
-				// if (this.minecraft.player != null)
-				// {
-				// float f = Mth.lerp(pPartialTicks, this.minecraft.player.oPortalTime,
-				// this.minecraft.player.portalTime);
-				//
-				// if (f > 0.0F && this.minecraft.player.hasEffect(MobEffects.CONFUSION) &&
-				// this.minecraft.options.screenEffectScale < 1.0F)
-				// {
-				// this.renderConfusionOverlay(f * (1.0F -
-				// this.minecraft.options.screenEffectScale));
-				// }
-				// }
-
-				if (!ClientDataHolder.viewonly) {
-					this.minecraft.gui.render(posestack1, partialTicks);
-				}
-
-//				if (this.minecraft.options.ofShowFps && !this.minecraft.options.renderDebug) { TODO
-//					Config.drawFps(matrixstack);
-//				} 
-
-//				if (this.minecraft.options.renderDebug) { TODO
-//					Lagometer.showLagometer(matrixstack, (int) this.minecraft.getWindow().getGuiScale());
-//				}
-
-				this.minecraft.getProfiler().pop();
-				RenderSystem.clear(256, Minecraft.ON_OSX);
-			}
-
-			if (this.guiLoadingVisible != (this.minecraft.getOverlay() != null)) {
-				if (this.minecraft.getOverlay() != null) {
-					LoadingOverlay.registerTextures(this.minecraft);
-
-					if (this.minecraft.getOverlay() instanceof LoadingOverlay) {
-						LoadingOverlay loadingoverlay = (LoadingOverlay) this.minecraft.getOverlay();
-						//loadingoverlay.update();
-					}
-				}
-
-				this.guiLoadingVisible = this.minecraft.getOverlay() != null;
```

Comment:
This was for showing the loading screen with optifine iirc. So it's good that it's gone, but you mentioned that you had a simple way to show the loading screen update? Is that included in a pr yet?